src/IncrementalChecker.ts

```diff
@@ -68,15 +71,19 @@ export class IncrementalChecker {
     }));
   }
 
-  static loadProgramConfig(configFile: string) {
-    return ts.parseJsonConfigFileContent(
+  static loadProgramConfig(configFile: string, compilerOptions: object) {
+    const parsed = ts.parseJsonConfigFileContent(
       // Regardless of the setting in the tsconfig.json we want isolatedModules to be false
       Object.assign(ts.readConfigFile(configFile, ts.sys.readFile).config, {
         isolatedModules: false
       }),
       ts.sys,
       path.dirname(configFile)
     );
+
+    parsed.options = { ...parsed.options, ...compilerOptions };
```

Comment:
I'm not sure if we can use object spread as it won't work on node 6 (and fork-ts-checker supports node 6).  https://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties

Can we switch to `Object.assign` please?

Reply:
Typescript will compile it down to `Object.assign` for you as long as your target is correct. It's a little tough for me to check as I'm on my phone right now but will double check tomorrow.

Reply:
If you use assign elsewhere in code base I don't mind changing either way!

Reply:
> Typescript will compile it down to Object.assign for you as long as your target is correct. It's a little tough for me to check as I'm on my phone right now but will double check tomorrow.

If that's the case then that's fine with me!